spec/support/blueberry_rails_helpers.rb

```diff
@@ -0,0 +1,43 @@
+require 'fileutils'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.